crates/nargo_cli/tests/test_data_ssa_refactor/unsigned_division/src/main.nr

```diff
@@ -0,0 +1,28 @@
+use dep::std;
+
+// Testing signed integer division: 
+//  7/3  = 2
+// -7/3  = -2
+// -7/-3 = 2
+//  7/-3 = -2
+fn main(mut x: i32, mut y: i32, mut z: i32) {
+    // 7/3 = 2
+    assert(x / y == z);
+
+    // -7/3 = -2
+    x = 0-x;
+    z = 0-z;
```

Comment:
can we not re-use x, y, z. minus_x, minus_y etc would be clearer

Reply:
Done